### ISSUE TITLE
Plugin doesn't work with 5.x versions of Logstash

### ISSUE THREAD

**Issue by RomainSNCF** [logstash-plugins/logstash-input-azure_event_hubs#4]
Hi,

This plugin doesn't work with 5.x versions of logstash.
function Gem.latest_spec_for doesn't exist in the rubygems.rb version distributed with this versions of Logstash.
So, a workaround is to comment all the content of the register function in vendor/bundle/jruby/1.9/gems/logstash-input-azureeventhub-0.9.8-java/lib/logstash/inputs/azureeventhub.rb and change by : 
user_agent = "logstash-input-azureeventhub/0.9.8"

Romain

**Comment by jakelandis** [comment]
@RomainSNCF - this plugin is getting a major overhaul at https://github.com/logstash-plugins/logstash-input-azure_event_hubs/pull/3 and will be compatible with 5.6+. It isn't released yet, but is getting close (docs, review, and minor clean up).

**Comment by jakelandis** [comment]
The re-implementation of this plugin has been released and tested to work with 5.6 ( should work with earlier 5.x versions too)

Note: This plugin is intended to superseded `logstash-input-azureeventhub`,  however since the name has changed to `logstash-input-azure_event_hubs` it will require explicit installation. All the options are different from the prior version and maybe found here https://github.com/logstash-plugins/logstash-input-azure_event_hubs/blob/master/docs/index.asciidoc until they show up in the standard docs. 

```
bin/logstash-plugin install logstash-input-azure_event_hubs
```

If there are issues with the new plugin and 5.x please open a new ticket with details.
